flipp_pomodoro/scenes/flipp_pomodoro_scene_config.c

```diff
@@ -0,0 +1,71 @@
+#include <gui/scene_manager.h>
+#include "../flipp_pomodoro_app.h"
+#include "flipp_pomodoro_scene.h"
+#include "../helpers/time.h"
+#include "../modules/flipp_pomodoro_settings.h"
+#include "../views/flipp_pomodoro_config_view.h"
+#include "../modules/flipp_pomodoro.h"
+#include <string.h>
+
+static void flipp_pomodoro_scene_config_on_save(void* ctx) {
+    FlippPomodoroApp* app = ctx;
+
+    FlippPomodoroSettings s_now;
+    flipp_pomodoro_view_config_get_settings(app->config_view, &s_now);
+    flipp_pomodoro_settings_save(&s_now);
+    // immediately apply to timers
+    flipp_pomodoro__apply_settings(&s_now);
+
+    scene_manager_next_scene(app->scene_manager, FlippPomodoroSceneTimer);
+}
+
+void flipp_pomodoro_scene_config_on_enter(void* ctx) {
+    FlippPomodoroApp* app = ctx;
+    app->paused_at_timestamp = time_now();
+
+    FlippPomodoroSettings s;
+    if(!flipp_pomodoro_settings_load(&s)) {
+        flipp_pomodoro_settings_set_default(&s);
```

Comment:
This seem to be a repeated construction. I see that it's solved inside of the method already.

TODO: remove repeated construction and make it default behaviour.